Makefile

```diff
@@ -1,4 +1,5 @@
 # dependencies
+# TODO this should be 3.46.0, but there are build errors
 SQLITE_VERSION = version-3.44.0
```

Comment:
I'm not sure why this fails for `3.46.0`. It seems like this error might be coming from our rust core
<img width="879" alt="image" src="https://github.com/user-attachments/assets/4196bc5e-4b01-41e5-b80c-88c1a327c8c7">